docs/src/content/integration/custom-events.mdx

```diff
@@ -111,6 +111,10 @@ Personally Identifiable Information (PII) includes any data that could identify
 
 ## Common Use Case Examples
 
+<Callout type="info">
+**Ready to view your custom events?** Once you've implemented custom event tracking, you can view and analyze your data in the [Events dashboard](/dashboard/events) to gain insights into user behavior.
```

Comment:
This link seems incorrect - we don't have a page called `/dashboard/events`.
Same goes for line 453